fast64_internal/f3d/f3d_bleed.py

```diff
@@ -479,9 +478,13 @@ def bleed_SPSetOtherMode(self, cmd_list: GfxList, cmd: GbiMacro, bleed_state: in
             else:
                 return cmd == self.default_othermode_L
 
-    # bleed if there are no tags to scroll and cmd was in last list
+    # Don´t bleed if the cmd tags are not the same (these are not hashed)
     def bleed_DPSetTileSize(self, cmd_list: GfxList, cmd: GbiMacro, bleed_state: int, last_cmd_list: GfxList = None):
-        return cmd.tags != GfxTag.TileScroll0 and cmd.tags != GfxTag.TileScroll1 and cmd in last_cmd_list
+        if cmd in last_cmd_list:
+            last_size_cmd = last_cmd_list[last_cmd_list.index(cmd)]
+            if last_size_cmd.tags == cmd.tags:
```

Comment:
If you have two materials with scrolls and the same cmd values but different scroll values they will bleed when they shouldn't. A scroll should never bleed.
This does prevent a cmd from bleeding after a scroll which is good and I have tested that that works. Just change this to something like, `cmd.tags != GfxTag.TileScroll0 and cmd.tags != GfxTag.TileScroll1 and cmd in last_cmd_list` and then do the same conditional you do here